lib/topological_inventory/persister/workflow.rb

```diff
@@ -216,6 +231,43 @@ def requeue_sweeping!
       def sweep_retry_count_limit
         100
       end
+
+      def link_data_to_refresh_state_part
+        refresh_state_part = ::RefreshStatePart.where(:uuid => refresh_state_part_uuid).first
+        if refresh_state_part.present?
+          ics = inventory_collections.select { |ic| ic.data.present? }
+          ics.each do |ic|
+            ic.data.each do |inventory_object|
+              inventory_object.data[:refresh_state_part_id] = refresh_state_part.id
+            end
+          end
+        end
+      end
+
+      # @param type [Symbol] :refresh_state_part | :refresh_state
+      def log_refresh_time_tracking(type)
+        msg = "Refresh tracking: "
+        msg += "State: #{refresh_state_uuid} "
```

Comment:
Any reason this can't be in the first `msg = "Refresh tracking: State: #{...}"` since there's no conditional here?

Reply:
emm, no, it's "historical" :D

Reply:
:laughing: